cloud_blobstore/__init__.py

```diff
@@ -1,6 +1,74 @@
 import typing
 
 
+class PagedIter:
+    """
+    Provide an iterator that will iterate over every object, filtered by
+    prefix and delimiter. Alternately continue iteration with token and
+    key (marker).
+    """
+
+    def __init__(
```

Comment:
the subclasses of PagedIter don't use this constructor nor do they follow its signature.  Am I missing something?